discord/member.py

```diff
@@ -149,7 +149,7 @@ class Member(discord.abc.Messageable, _BaseUser):
     ----------
     joined_at: Optional[:class:`datetime.datetime`]
         A datetime object that specifies the date and time in UTC that the member joined the guild for
-        the first time. In certain cases, this can be ``None``.
+        the first time. If the member left and rejoined the guild, this will be the latest date. In certain cases, this can be ``None``.
```

Comment:
Adding this statement makes the docstring conflict with itself - either it's the time the member joined for the first time (it isn't), or it's the most recent join.

It would make more sense to just remove anything related to 'the first time the member joins' - since I'm not even sure if Discord themselves records such information.

Reply:
oof, you are right. also, already fixed it.